### ISSUE TITLE
About the experiment

### ISSUE THREAD

**Issue by wanwan-na** [YonghengSun1997/MSCA-Net#7]
Hello, I would like to ask whether your experiment (mean ± standard deviation) is The test output result, for example, The ISIC mean dice: 0.9239; The ISIC dice std: 0.1149 As a result? Is it 0.9239+-0.1149?
![1](https://github.com/YonghengSun1997/MSCA-Net/assets/93858453/0c7c4e54-b4f5-4d48-869a-c2b193d21c40)

**Comment by YonghengSun1997** [comment]
Yes. For each fold from five folds, we calculate the mean result on test set of each fold. Then the reported mean and std is calculated on these five results.

**Comment by wanwan-na** [comment]
Would you like to ask if the dice value is greater than 1 when 0.9239+-0.1149 is added, is this also normal?

**Comment by YonghengSun1997** [comment]
What do you mean by "the dice value is greater than 1 when 0.9239+-0.1149 is added"?

**Comment by wanwan-na** [comment]
If the final dice is 0.9239+-0.1149, 0.9239+0.1149 is greater than 1, but shouldn't the dice value be between 0 and 1?

**Comment by YonghengSun1997** [comment]
I can't find the "0.9239+-0.1149" result. I guess "0.9239+-0.1149" may be " (92.39 + 0.1149) %? The std is calculated on five results from five fold (std is calculated on five scalars). The five fold results is rather stable, and the std of these five results is rather small.

**Comment by wanwan-na** [comment]
I averaged the dice value and std obtained from the cross-verification of 50% fold respectively to get the final result. Is it correct to write it in yellow at the end?
![2](https://github.com/YonghengSun1997/MSCA-Net/assets/93858453/e4c1164d-8510-44e1-a3dc-19f82800cd21)

**Comment by YonghengSun1997** [comment]
Yes, I reported mean and std of 5 folds, like your results in yellow.

**Comment by YonghengSun1997** [comment]
So ”0.9239+-0.1149“ is your fold 1 results. I think it is normal, because some cases is hard to segmentation, and have low dices （such as below 0.65）, which causes big std. Actually, the ISIC challenge treat iou below 0.65 as 0 when evaluation.

**Comment by wanwan-na** [comment]
Ok, thank you very much for your answer
